lib/unleash/toggle_fetcher.rb

```diff
@@ -53,7 +53,7 @@ def fetch
       headers['Content-Type'] = 'application/json'
       headers['If-None-Match'] = self.etag unless self.etag.nil?
 
-      request = Net::HTTP::Get.new(uri.request_uri, headers)
+      request = Net::HTTP::Get.new(uri.request_uri + "?instance_id=#{Unleash.configuration.instance_id}", headers)
```

Comment:
These should be added as headers and not as query params. Proper headers to add to all unleash requests are: 
- `UNLEASH-INSTANCEID`
- `UNLEASH-APPNAME`

Reply:
@ivarconr So the proper way to set client id is

```
# config/initializers/unleash.rb

Unleash.configure do |config|
  config.url      = 'http://unleash.herokuapp.com/api'
  config.app_name = Rails.application.class.parent.to_s
  config.custom_http_headers = {
      UNLEASH-INSTANCEID: 'xxxxxxxx', # Set instance id
      UNLEASH-APPNAME: 'xxxxxxxx', # Set App name 
  }
end

UNLEASH = Unleash::Client.new
```

?

If so, does it make sense to document it on README.md as users can set instance id/app name in this way?

Reply:
yes, and yes it would of course make sense to document this better.

Reply:
I will fix this in a separate branch/PR. I was not aware of these headers.

It should not be necessary afterwards to manually configure those as `custom_http_headers`.

Reply:
@rarruda Yes, feel free to ignore this PR. I have already confirmed that the above config can work with GitLab.